src/components/organisms/about/about.html

```diff
@@ -0,0 +1,20 @@
+<div class="organisms-about container-fluid">
```

Comment:
```suggestion
<div class="organisms-about container">
```

`container` を使って左右の余白を再現してください。

![image](https://user-images.githubusercontent.com/83479175/218262679-7b2951a6-d5e7-4d08-946c-a0e0a20a4c05.png)
`.container` を使うと、左右の余白はこんな感じで再現できます。

Reply:
対応しました!

0576e250a28af08f4a9b077ae50e425ac3420219